src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const REVERSE_ROBOT = {};
+  const VALUES = Object.values(robot);
+  const KEYS = Object.keys(robot);
+  const REMOVE_DUBLICATE_VALUES = new Set(VALUES);
+
+  if ([...REMOVE_DUBLICATE_VALUES].length !== VALUES.length) {
+    return null;
+  }
```

Comment:
The check for duplicates is correct, but it could be done more efficiently. Instead of creating a new array from the `Set` and comparing its length to the original array's length, you could simply check if the `Set`'s size is less than the original array's length. This would be more efficient and easier to understand.